mint.json

```diff
@@ -57,9 +57,10 @@
     {
       "group": "Devices",
       "pages": [
-        "api-reference/endpoint/get",
+        "api-reference/endpoint/get-devices",
         "api-reference/endpoint/send-command",
-        "api-reference/endpoint/edit"
+        "api-reference/endpoint/edit",
+        "api-reference/endpoint/get-single-device"
```

Comment:
From my experience, I've the same pattern across many APIs. That said, since the difference of an 's' may be too subtle, I just wanted to make it absolutely clear that this endpoint is to fetch a single device. I don't feel too strongly about it. Let me know what you think

Reply:
Totally! It's great to strive for clarity. I think it is clear enough without the 's' alone, especially since you can see upon clicking it that it requires a single device id. I am also thinking about the naming in the context of the api endpoints as a whole. We will eventually have endpoints like GET Configuration, GET App, GET Wifi Network etc. and we won't want to have the extra 'single' indicator in there because it will make the list as a whole more wordy/less scannable.